backend/mach.mli

```diff
@@ -27,6 +27,9 @@ type integer_comparison =
     Isigned of Cmm.integer_comparison
   | Iunsigned of Cmm.integer_comparison
 
+type atomic_integer_operation =
+  Ifetchadd | Ifetchsub | Ifetchand | Ifetchor | Ifetchxor | Icompareandswap
```

Comment:
It would be better to use  the same names as in `Cmm.atomic_op`.

Reply:
Changed